tests/c/funcptrarg_hasir.c

```diff
@@ -1,4 +1,4 @@
-// Error:
+// # Error:
```

Comment:
I don't understand what we've done to this test: why is so much of it commented out?

Reply:
I think this is an out-of-date version of the file, I removed the comment in the latest commit: https://github.com/Pavel-Durov/yk-fork/blob/software-tracer/tests/c/funcptrarg_hasir.c

Originally I was adding comments to keep track of the failing tests.